pkg/env/env.go

```diff
@@ -0,0 +1,27 @@
+package env
+
+import (
+	"fmt"
+	"os"
+)
+
+const (
+	// EnvVarPodNamespace is the process environment variable representing the
+	// POD_NAMESPACE env var which is populated via the Downward API.
+	EnvVarPodNamespace = "POD_NAMESPACE"
+)
+
+var (
+	podNamespace string
+)
+
+func init() {
+	podNamespace = os.Getenv(EnvVarPodNamespace)
+	if podNamespace == "" {
```

Comment:
Is this going to be `giantswarm` in the unique app, right?

Reply:
Yes it will be giantswarm but because the unique flag is true we don't set the namespace in the operatorkit controller. 

This is so we can process the new app-operator app CR in the cluster namespace.